wordpress/cloud-config.yaml

```diff
@@ -78,6 +78,7 @@ runcmd:
     - mysql -e "CREATE USER wordpress@localhost IDENTIFIED BY '$PASSWORD';"
     - mysql -e "GRANT ALL PRIVILEGES ON wordpress.* TO 'wordpress'@'localhost';"
     - mysql -e "FLUSH PRIVILEGES;"
+    - chmod +x /home/{{users.0.username}}
```

Comment:
As of Ubuntu 22.04 the user dir has 750 permissions by default rather than 755.

Reply:
That poses an interesting problem indeed.
What's harder to grasp - all newly created users must be adapted manually or change the default umask for the system (deviating from upstream default). I am not sure what path to choose really. From a support staff perspective it's probably better to not change the default and have "check user dir permissions" as the first bullet point in troubleshooting.